src/main/java/org/wso2/carbon/module/swiftiso20022/MT940FormatValidator.java

```diff
@@ -98,30 +98,30 @@ public void connect(MessageContext messageContext) {
      * Method to validate the MT940 format.
      *
      * @param messageContext  Message context
-     * @param lines           Payload lines
-     * @return                Error model
+     * @param lines           Payload lines containing the Mt940 fields
+     * @return        Validation Result with error details if there is an error, else empty Validation Result
      */
-    private ErrorModel validateMT940(MessageContext messageContext, String[] lines) {
+    private ValidationResult validateMT940(MessageContext messageContext, String[] lines) {
         Map<String, Object> extractFields = extractFields(lines);
```

Comment:
Shall we change the variable name to extractedFields

Reply:
Fixed in https://github.com/wso2-extensions/mediation-swift-iso20022-module/pull/6/commits/7f2058c938da71ec80755d296fd2d3d67541e678